CMakeLists.txt

```diff
@@ -1438,6 +1438,71 @@ if(WIN32)
   target_link_libraries(mixxx-lib PUBLIC FFTW::FFTW)
 endif()
 
+# libdjinterop (enabled only on UNIX for now)
+cmake_dependent_option(DJINTEROP "Use djinterop library" ON "UNIX;NOT APPLE" OFF)
```

Comment:
This option should be disabled by default and explicitly enabled in the CI builds.

Reply:
Changed.